### PR TITLE
Partial improvement to the model fitting chapter

### PR DESCRIPTION
**Pull request by francisco-dlp**
While reviewing #2441 I noticed that the information in the  model fitting chapter of the UG was presented in a way that was hard to follow for new users. Although there is still a lot of room for improvement, I think that this is a first step.

### CODE REVIEW

**Comment by francisco-dlp** [comment]
For those who haven't noticed it, now the documentation is automatically built and served by pypi at each PR.

**Comment by ericpre** [comment]
I did notice this and having a preview just "one click away" is very nice! This would deserve a mention in the dev guide! ;)

**Comment by tjof2** [comment]
Agreed this is a nice first step! LGTM

File: doc/user_guide/model.rst [hunk 1]
```diff
@@ -241,6 +237,24 @@ the following template to suit your needs:
             """
             return x
 
+Define components from a fixed-pattern
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :py:class:`~._components.scalable_fixed_pattern.ScalableFixedPattern`
+component enables fitting a pattern (in the form of a
+:py:class:`~._signals.signal1d.Signal1D` instance) to data by shifting 
+(:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.xshift`)
```
Comment (ericpre) [inline doc/user_guide/model.rst]: The link to this attribute doesn't work?

File: doc/user_guide/model.rst [hunk 2]
```diff
@@ -452,8 +446,32 @@ terminal view with the argument ``fancy=False``. One can also filter for only ac
...
-The current coordinates can be either set by navigating the :py:meth:`~.model.BaseModel.plot`, or specified by
-pixel indices in ``m.axes_manager.indices`` or as calibrated coordinates in
+.. _Component.print_current_values:
+
+The current values of a particular component can be printed using the
+:py:attr:`~.component.Component.print_current_values()` method. The
+IPython display function elegantly presents it using HTML
+and allows for correct copying and pasting into Excel spreadsheets.
+Alternatively, a simpler form can be shown by setting the
+``fancy`` argument to ``False``
```
Comment (ericpre) [inline doc/user_guide/model.rst]: This is repetitive, since this has been mentioned just before.

**Review by ericpre — COMMENTED** [review]
Two suggestions for improvement.

File: doc/user_guide/model.rst (hunk 1 above)
Comment (francisco-dlp) [reply doc/user_guide/model.rst]: Actually there was a typo there that I've just fixed. However, the links can't work because the attributes are created by the constructor.

File: doc/user_guide/model.rst [hunk 3]
```diff
@@ -241,6 +237,24 @@ the following template to suit your needs:
...
+:py:class:`~._signals.signal1d.Signal1D` instance) to data by shifting 
+(:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.shift`)
+and
+scaling it in the x and y directions using the
+:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.xscale`
+and
+:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.yscale`
+parameters respectively.
+
+``xscale`` and ``yscale`` parameters. The position of the pattern can also be adjusted
```
Comment (ericpre) [inline doc/user_guide/model.rst]: There is a repetition here too!
https://hyperspy--2455.org.readthedocs.build/en/2455/user_guide/model.html#define-components-from-a-fixed-pattern

Comment (francisco-dlp) [reply doc/user_guide/model.rst]: Done, thanks!
